docs/howto/pom.xml

```diff
@@ -88,18 +91,6 @@
       </resource>
     </resources>
     <plugins>
-      <plugin>
```

Comment:
Why was this removed?

Reply:
Because each howto defines `thorntail-maven-plugin` on its own -- unfortunately, using a different `<id>` of the `<execution>`. So the Thorntail Maven plugin's `package` goal used to be invoked twice.

Reply:
Makes sense, thanks